Package.swift

```diff
@@ -3,6 +3,10 @@ import PackageDescription
 
 let package = Package(
     name: "Endpoints",
+    platforms: [
+        .macOS(.v10_15),
+        .iOS(.v13)
+    ],
```

Comment:
I think it would be better to not specify platforms so we implicitly support all of them.
To check if async support ist available we should use conditionals. See here for examples: https://swiftindepth.com/articles/async-await-modules/